spec/services/slack_service_spec.rb

```diff
@@ -11,4 +11,15 @@
       subject.post_message('london', 'cheers!')
     end
   end
+
+  describe "Posting a message to a cheered user" do
+    let (:client) { double(:slack_client) }
+    subject { described_class.new(client) }
```

Comment:
As these `let` and `subject` are the same as lines 6-7 we could move this to the top of the outer `describe` and remove those.